test/library/standard/Random/RandomStreamInterface/getNextTestHalts.good

```diff
@@ -0,0 +1 @@
+getNextTestHalts.chpl:11: error: halt reached - Cannot generate random numbers within empty range: [2, 1]
```

Comment:
Needs a skipif or suppressif for --fast (i.e. test/types/range/deitz/test_range_iterate_error2.suppressif)